extensions/amp-story-player/validator-amp-story-player.protoascii

```diff
@@ -42,4 +42,40 @@ descendant_tag_list {
   tag: "A"
   tag: "SPAN"
   tag: "I-AMPHTML-SIZER"  # Only allowed when document is transformed AMP
+  tag: "IMG"
+}
+tags: {
```

Comment:
From my searching, this will be the first usage of `img` outside of `validator-main.protoascii` which has one for standard AMP and several for Transformed AMP. Would you mind updating the note at `4.7 Embedded Content` that it is also defined in this file for AMP Story? That or add it under `4.7.1` instead.

Reply:
Added the note 👍